### PR TITLE
nsqadmin: missing channel from topic list

### PR DESCRIPTION
**Pull request by jehiah**
the api endpoint `/api/topics/:topic` seems to be missing a channel that i see in `/stats` output from nsqd's. I'm not sure what the bug is yet, but this causes it to be missing from the nsqadmin UI at `/topics/:topic` even though it shows properly if you manually navigate to `/topics/:topic/:channel`.

### CODE REVIEW

**Comment by jehiah** [comment]
RFR @mreiferson @ploxiln 

I was able to reproduce this bug with the following setup

Node A: topic: T channels a,b,c
Node B: topic: T channels a.  

With that setup the count was previously 3 on both nodes, and depending on the order `/stats` responses completed you would see all three channels listed on the topics page, or just one.

File: internal/clusterinfo/types.go [hunk 1]
```diff
@@ -115,8 +115,8 @@ func (t *TopicStats) Add(a *TopicStats) {
 	if a.Paused {
 		t.Paused = a.Paused
 	}
-	found := false
 	for _, aChannelStats := range a.Channels {
+		found := false
```
Comment (ploxiln) [inline internal/clusterinfo/types.go]: nice fix 👍

File: nsqadmin/static/js/views/topic.hbs [hunk 2]
```diff
@@ -81,7 +81,7 @@
             {{#if ../graph_active}}
                 <td class="bold rate" target="{{rate "topic" node topic_name ""}}"></td>
             {{/if}}
-            <td>{{commafy ../channels.length}}</td>
+            <td>{{commafy this/channels.length}}</td>
```
Comment (ploxiln) [inline nsqadmin/static/js/views/topic.hbs]: this is greek to me :)
I wonder if "this/" is not needed and just "channels.length" would suffice?

Comment (jehiah) [reply nsqadmin/static/js/views/topic.hbs]: I'm not sure... I confused myself by working in the wrong part of the template for a while, but there is a global `channels` var so this is at least more explicit.

File: internal/clusterinfo/types.go (hunk 1 above)
Comment (jehiah) [reply internal/clusterinfo/types.go]: it only took me about a hundred printf's to find it =)

File: nsqadmin/static/js/views/topic.hbs (hunk 2 above)
Comment (ploxiln) [reply nsqadmin/static/js/views/topic.hbs]: fair enough :) no objection

**Pull request merged**